salsa_dancing_molecules/unit_test/test_msd.py

```diff
@@ -0,0 +1,22 @@
+"""Unit test for mean_square_displacement.py."""
+
+import pytest
+from ..mean_square_displacement import calculate_msd
+from unittest.mock import Mock
+
+
+atoms = Mock()
+atoms2 = Mock()
+atoms.__len__ = Mock(return_value=2)
+atoms.get_cell_lengths_and_angles.return_value = [10, 10, 10]
+atoms.get_positions.return_value = [[1, 1, 1], [2, 2, 2]]
+atoms2.get_positions.return_value = [[9, 9, 9], [3, 3, 3]]
+configs = [atoms, atoms2]
+
+
+@pytest.mark.filterwarnings("ignore: Polyfit may be poorly conditioned")
```

Comment:
You don't need this suppression in your code since you, as far as I can tell, do not use any fitting. I assume that this was copied from the lattice constant test and can be removed.